common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4782,6 +4788,12 @@
     ],
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_DATETIME_UNIT_ADDITION" : {
```

Comment:
Added extra error class to cover the specific case and to get rid of message as a parameter for error class. cc @srielau @cloud-fan fyi